functions/exercise4/scriptSpect.js

```diff
@@ -0,0 +1,8 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals validatePhrase*/
+
+describe("Check if", function () {
```

Comment:
describe should information about piece of the code (function) which is tested by tests described below